src/backend/api/routes/auth/github.route.py

```diff
@@ -93,6 +95,21 @@ async def verify(code: str, session: SessionStorage = fastapi.Depends(SessionSto
     except pydantic.ValidationError:
         raise fastapi.HTTPException(fastapi.status.HTTP_400_BAD_REQUEST, detail=data.get("error_description"))
 
-    session.set("token", data.access_token)
-
-    return session.toRedirectResponse(url="/")
+    with createLocalSession() as connection:
+
+        session = connection\
+            .query(dbm.Session)\
+            .filter(dbm.Session.access_token == data.access_token)\
+            .one_or_none()
+        if not session:
```

Comment:
```suggestion
        if session == None:
```

Reply:
ich finde if not session ehrlich gesagt schöner, habe aber wenig python erfahrung, daher kann ich das schwer beurteilen

Reply:
```suggestion
        if session is None:
```
"not" geht auch, aber da passiert implizites type-casting. Das heißt das der Integer 0 zum beispiel auch als "False" interpretiert wird.
Da wir aber genau "None" erwarten kann man hier spezifischer sein. (Ich habe heute von Robin gelernt das "is" besser als "==" ist)